lib/std/http/status.zig

```diff
@@ -1,182 +0,0 @@
-//! HTTP Status
-//! https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
-
-const std = @import("../std.zig");
-
-pub const Status = enum(u10) {
-    @"continue" = 100, // RFC7231, Section 6.2.1
-    switching_protocols = 101, // RFC7231, Section 6.2.2
-    processing = 102, // RFC2518
-    early_hints = 103, // RFC8297
-
-    ok = 200, // RFC7231, Section 6.3.1
-    created = 201, // RFC7231, Section 6.3.2
-    accepted = 202, // RFC7231, Section 6.3.3
-    non_authoritative_info = 203, // RFC7231, Section 6.3.4
-    no_content = 204, // RFC7231, Section 6.3.5
-    reset_content = 205, // RFC7231, Section 6.3.6
-    partial_content = 206, // RFC7233, Section 4.1
-    multi_status = 207, // RFC4918
-    already_reported = 208, // RFC5842
-    im_used = 226, // RFC3229
-
-    multiple_choice = 300, // RFC7231, Section 6.4.1
-    moved_permanently = 301, // RFC7231, Section 6.4.2
-    found = 302, // RFC7231, Section 6.4.3
-    see_other = 303, // RFC7231, Section 6.4.4
-    not_modified = 304, // RFC7232, Section 4.1
-    use_proxy = 305, // RFC7231, Section 6.4.5
-    temporary_redirect = 307, // RFC7231, Section 6.4.7
```

Comment:
nit

[306 is reserved](https://www.rfc-editor.org/rfc/rfc9110.html#name-306-unused)
since it's a non-exhaustive enum, it shouldn't matter?